tools/hivelvet

```diff
@@ -216,6 +216,39 @@ enable_installer() {
       sed -i '/config.extension =/ c config.extension = -install' "$BASEDIR/../hivelvet-backend/app/config/config-$ENVIRONMENT.ini"
       printf "%s %s [✔ - DONE]\n"
     fi
+    #c cmd to update .ant-card-body and .presets-grid css classes
```

Comment:
It should be possible to load 2 different stylesheets depending on the context. Or paramaterised style loading.